src/examples/forms/range/RangeValidation.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Field, Label, Range, Message } from '@zendeskgarden/react-forms';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+type validationTypes = 'success' | 'warning' | 'error';
+
+const valType: Record<validationTypes, string> = {
+  success: 'Growing all the time',
+  warning: 'Growing regularly',
+  error: 'Growing slowly'
+};
+
+const Example = () => {
+  const [validation, setValidation] = useState<validationTypes>('warning');
+
+  const readValue = (e: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
Can this be renamed to `onChange`? The common variable names for React event handlers usually reflect their name or is a variant of `onSomeEvent`. Renaming this would make the name consistent with our codebase too.

Alternatively, we can inline this function. That way you don't have to worry about naming and TypeScript type inference infers the type of `e`.